### ISSUE TITLE
Качает не все файлы

### ISSUE THREAD

**Issue by sh2ka** [genaev/vmd#18]
Возможно, это связано с новой версией VK. Возможно, они блокируют скачивание некоторых записей. Однако, я точно не знаю. В общем часть записей даже не отображаются в списке, который выдаёт vmd.

**Comment by genaev** [comment]
Да, есть такая проблема. Я тоже столкнулся. Однако почему так происходит сказать не могу. Возможно, Вы правы по поводу обновления vk.
